scripts/globals/spells/trust/ferreous_coffin.lua

```diff
@@ -1,5 +1,12 @@
 -----------------------------------
 -- Trust: Ferreous Coffin
+-- Auto Refresh II (Cleric's Bliaut +2). HP-10%, MP+35%
```

Comment:
Interesting implication of visible gear effecting trusts' stats. I wonder if there are more? Some gear like AF should be more easily verifiable, just worth a thought when implementing trusts. Too bad Ferreous Coffin doesn't cast regen, we could verify Regen potency